experimental/addons/spark-kubernetes-operator/README.md

```diff
@@ -0,0 +1,295 @@
+             SSH: vela exec spark-app-v1 -n spark-cluster
+         Logging: vela logs spark-app-v1 -n spark-cluster
+      App status: vela status spark-app-v1 -n spark-cluster
+        Endpoint: vela status spark-app-v1 -n spark-cluster --endpoint
+Application spark-cluster/spark-app-v1 applied.
+```
+
+3. Then, there are serval ways to check the status(or detail information) of the Spark applicaiton:
+
+option 1:
+
+```
+vela status spark-app-v1 -n spark-cluster
+About:
+
+  Name:      	spark-app-v1
+  Namespace: 	spark-cluster
+  Created at:	2023-02-27 16:54:37 +0800 CST
+  Status:    	running
+
+Workflow:
+
+  mode: DAG-DAG
+  finished: true
+  Suspend: false
+  Terminated: false
+  Steps
+  - id: vfgjkrxvih
+    name: spark-workload-component
+    type: apply-component
+    phase: succeeded
+
+Services:
+
+  - Name: spark-workload-component
+    Cluster: local  Namespace: spark-cluster
+    Type: spark-workload
+    Healthy
+    No trait applied
+```
+
+option 2:
+
+```
+kubectl get sparkapplications -n spark-cluster
+NAME           STATUS    ATTEMPTS   START                  FINISH       AGE
+my-spark-app   RUNNING   1          2023-02-27T08:54:40Z   <no value>   2m33s
+```
+
+option 3:
+
+```
+kubectl describe sparkapplication my-spark-app -n spark-cluster
+Name:         my-spark-app
+Namespace:    spark-cluster
+Labels:       app.oam.dev/app-revision-hash=4e5592aea53a5961
+              app.oam.dev/appRevision=spark-app-v1-v1
+              app.oam.dev/cluster=local
+              app.oam.dev/component=my-spark-application-component
+              app.oam.dev/name=spark-app-v1
+              app.oam.dev/namespace=spark-cluster
+              app.oam.dev/resourceType=TRAIT
+              app.oam.dev/revision=
+              oam.dev/render-hash=640a3298d803274e
+              trait.oam.dev/resource=spark
+              trait.oam.dev/type=AuxiliaryWorkload
+Annotations:  app.oam.dev/last-applied-configuration:
+                {"apiVersion":"sparkoperator.k8s.io/v1beta2","kind":"SparkApplication","metadata":{"annotations":{"app.oam.dev/last-applied-time":"2023-02...
+              app.oam.dev/last-applied-time: 2023-02-27T16:54:37+08:00
+              oam.dev/kubevela-version: v1.7.0
+API Version:  sparkoperator.k8s.io/v1beta2
+Kind:         SparkApplication
+Metadata:
+......
+```
+
+option 4:
+
+```
+kubectl get app spark-app-v1 -n spark-cluster -oyaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+......
+```
+
+4. Show the service of spark application via this command:
+
+```
+kubectl get svc -n spark-cluster
+NAME                                       TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                      AGE
+my-spark-app-c58a1c869214bfe5-driver-svc   ClusterIP   None             <none>        7078/TCP,7079/TCP,4040/TCP   19m
+my-spark-app-ui-svc                        ClusterIP   xx.xx.xx.xx      <none>        4040/TCP                     19m
```

Comment:
vela has a command:

```
vela status spark-app-v1 -n spark-cluster --endpoint
```

It can provide the endpoint for you automatically when discover a service created, you might need to define the resource topology for CRD, to describe the relationship between CRD and service: https://kubevela.net/docs/reference/topology-rule .

Reply:
Good suggestion. Will try to refactor it later.